app/ui-utils/client/lib/openRoom.js

```diff
@@ -1,9 +1,9 @@
 import { Meteor } from 'meteor/meteor';
 import { Tracker } from 'meteor/tracker';
 import { Blaze } from 'meteor/blaze';
+import { BlazeLayout } from 'meteor/kadira:blaze-layout';
```

Comment:
Is it really necessary to move this import?

Reply:
PR #621

Reply:
PR Assistify #621